tools/bindsnoop.py

```diff
@@ -389,6 +393,16 @@
     print(bpf_text)
     exit()
 
+# disable sys.stdout when --json is passed
+stdout = sys.stdout
+if args.json:
+    sys.stdout = open("/dev/null", 'w', buffering=1)
+
+    def stubprintb(s, file=sys.stdout, nl=1):
+        return
+
+    printb = stubprintb
```

Comment:
Is it possible to factor this in a function of `bcc.containers`? I am not sure if it works, maybe the function would have to be called before the `printb` import (`from bcc.utils import printb) but after the `bcc.containers` import...

Reply:
I created a `disable_output` helper in the bcc.utils package. The caller still has to assign the return value to `pritnb` but I think it's ok.